config/initializers/time_formats.rb

```diff
@@ -0,0 +1,2 @@
+Time::DATE_FORMATS[:day_month_year] = '%d-%m-%Y' # 04-11-2021
```

Comment:
Use the view/decorator layer instead

Reply:
I have updated this to make use of the existing date helper (renamed what is in here to use for citation as well)